azure-pipelines.yml

```diff
@@ -22,10 +22,10 @@ jobs:
       Python39:
         python.version: '3.9'
   steps:
-  - script: pip install -r continuous-integration\requirements-build-and-test.txt"
+  - script: pip install --requirement continuous-integration\python-requirements\build.txt
```

Comment:
The requirement needed here is not "build". Here we are outside docker and "build" happens inside.

Maybe split the file in `ci.txt` and `build.txt`.